wolox-books/src/styles.scss

```diff
@@ -0,0 +1 @@
+/* You can add global styles to this file, and also import other style files */
```

Comment:
You should delete this file if you aren't going to use it.

Reply:
Deleted ❌😅